CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.0.0] - revision: 2023-01-24
+
+- Added the following endpoints (which were previously in our Beta API/SDKs):
+  - Data Privacy
+  - All Tags endpoints, as well as the following related resource-specific endpoints:
+    - Get Flow Tags
+    - Get List Tags
+    - Get Segment Tags
+- Reworked the response object from api
+  - instead of only returning the response payload, the status code and headers are now also returned
+  - original payload items are now being `body`. to get the `id` by calling `response.body.data.id` instead `response.data.id` from previous releases
```

Comment:
Maybe something like the following to be super clear on how to get this?
```suggestion
  - The API response payload is now nested under `body` within the API response returned by SDK methods. You can get the `id` of the returned resource by calling `response.body.data.id` instead of `response.data.id` from previous releases. You can get response headers and status with `response.headers` and `response.status` respectively.
```